plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/ConfluentSchemaRegistryTableDescriptionSupplier.java

```diff
@@ -272,17 +273,23 @@ private TopicAndSubjects parseTopicAndSubjects(SchemaTableName encodedSchemaTabl
     private static boolean isValidSubject(String subject)
     {
         requireNonNull(subject, "subject is null");
-        return subject.endsWith(VALUE_SUFFIX) || subject.endsWith(KEY_SUFFIX);
+        return (subject.endsWith(VALUE_SUFFIX) && subject.length() > VALUE_SUFFIX.length())
```

Comment:
Can we `String#match` so that we could pass a regex that matches both the suffix and length ?

Reply:
that would require yet another constant, so would rather not to